src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,19 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
-);
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+
+    <section className="TodoList">
+      {todos.map((todo) => (
+        <TodoInfo key={todo.id} todo={todo} />
+      ))}
+    </section>
+  );
+};
```

Comment:
```suggestion
export const TodoList: React.FC<Props> = ({ todos }) => (
  <section className="TodoList">
    {todos.map((todo) => (
      <TodoInfo key={todo.id} todo={todo} />
    ))}
  </section>
);
```
if you only return in the component you may write it in this way